charts/catalog/values.yaml

```diff
@@ -89,6 +89,8 @@ controllerManager:
   serviceAccount: service-catalog-controller-manager
   # Controls whether the API server's TLS verification should be skipped.
   apiserverSkipVerify: true
+  # Whether the controller will expose metrics on /metrics
+  enablePrometheusScrape: false
```

Comment:
If this changes the behavior of the controller itself, we might call this `enableMetrics` - as it's worded now it sounds like it might change the behavior of prometheus instead of the controller.

Reply:
as detailed in https://github.com/kubernetes-incubator/service-catalog/pull/1608/files/199051332bca08be72e229de241a3ba5f21c3bd1#r154350489, this is used to set the deployment annotation which indicates the /metrics endpoint is available for collecting Prometheus metrics.

Perhaps a feature gate should be used to drive this instead - if the feature gate is off, the scrape would be disabled and we would skip registering the Prometheus http handler (https://github.com/kubernetes-incubator/service-catalog/pull/1608/files/199051332bca08be72e229de241a3ba5f21c3bd1#diff-eedddd33aeb1258962217ab4c61c82e0R65)